themes/src/main/resources/theme/base/admin/resources/js/services.js

```diff
@@ -349,18 +349,8 @@ module.factory('RegisterRequiredAction', function($resource) {
     });
 });
 
-module.factory('RealmLDAPConnectionTester', function($resource, $httpParamSerializer) {
```

Comment:
Again, you seem to break things when rebasing with latest master. You removed recently added changes and replaced with your local ones.